Packs/CortexResponseAndRemediation/ReleaseNotes/1_1_46.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Suspicious LDAP search query
+
+Updated the Suspicious LDAP search query playbook by removing the CMD prevalence check, replaced the old CommandLine-Analysis playbook with the new script an updated conditions.
```

Comment:
I don't understand this sentence - 
is the playbook being replaced with a script? does the script have updated conditions? or an updated script is being used in a new CommandLine-Analaysis playbook?